frontend/src/app/components/channel/large/large.controller.ts

```diff
@@ -10,4 +12,22 @@ export class LargeChannelController extends SingleUserChannelController {
     }
   }
 
+  onDragUserFrom(data: IDraggedData) {
+    if (data.destinationType === 'search') {
+      return;
+    }
+    this.users.some((user: IUser, index: any) => {
```

Comment:
Please write comments with story about what are you doing or simplify code so it can be understood simply.